Source/Core/VideoCommon/VideoConfig.cpp

```diff
@@ -261,6 +261,11 @@ void VideoConfig::VerifyValidity()
       iStereoMode = 0;
     }
   }
+  if (!backend_info.bSupportsBBox && g_ActiveConfig.bBBoxEnable)
+  {
+    PanicAlertT("Your graphics card does not support bounding box emulation this game requires. "
+                "You will experience bugs and crashes. This is not a Dolphin bug.");
```

Comment:
I would suggest something like "Bounding box is supposed to be emulated, but your graphics card or its drivers does not support it. Bounding box emulation will be skipped, which leads to problems in certain games". But that's just a draft.

(I was thinking of having "Bounding box is enabled" at the beginning, but that would be confusing since you actually _disable_ a setting to get bbox emulation)